Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -326,6 +326,45 @@ void IConfigurableHttpClientInitializer.Initialize(ConfigurableHttpClient httpCl
         /// </summary>
         public ICredential UnderlyingCredential => credential;
 
+        /// <summary>Creates a <c>GoogleCredential</c> wrapping a <see cref="ImpersonatedCredential"/>.</summary>
+        /// <param name="TargetPrincipal">The service account to impersonate.</param>
+        /// <param name="Delegates">The chained list of delegates.</param>
```

Comment:
I suggest this should probably be renamed to something like `delegateAccounts`, and the comment needs to be clearer. Bear in mind that "delegate" has a very specific meaning in C# which has nothing to do with what we're talking about here.

Reply:
done